rest/making-calls-sip/example-2/example-2.4.x.rb

```diff
@@ -8,8 +8,8 @@
 @client = Twilio::REST::Client.new account_sid, auth_token
 
 call = @client.account.calls.create(:url => "http://www.example.com/sipdial.xml",
-    :to => "sip:kate@example.com",
-    :from => "Jack",
-    :sip_auth_password => "secret",
-    :sip_auth_username => "jack")
+  :to => "sip:kate@example.com",
```

Comment:
ditto - Seems like you should apply the same changes to all the examples that has multiple params.